build/scripts/build.ps1

```diff
@@ -338,15 +338,33 @@ function Redirect-Temp() {
     ${env:TMP} = $temp
 }
 
+function List-BuildProcesses() {
+    Get-Process -Name "msbuild" -ErrorAction SilentlyContinue | Out-Host
+    Get-Process -Name "vbcscompiler" -ErrorAction SilentlyContinue | Out-Host
+}
+
+function List-VSProcesses() {
+    Get-Process -Name "devenv" -ErrorAction SilentlyContinue | Out-Host
+    List-BuildProcesses
+}
+
 # Kill any instances VBCSCompiler.exe to release locked files, ignoring stderr if process is not open
 # This prevents future CI runs from failing while trying to delete those files.
 # Kill any instances of msbuild.exe to ensure that we never reuse nodes (e.g. if a non-roslyn CI run
 # left some floating around).
 function Stop-BuildProcesses() {
-    Get-Process msbuild -ErrorAction SilentlyContinue | kill 
+    Get-Process msbuild -ErrorAction SilentlyContinue | kill
     Get-Process vbcscompiler -ErrorAction SilentlyContinue | kill
 }
 
+# Kill any instances of devenv.exe to ensure VSIX install/uninstall works in future runs and to ensure
+# that any locked files don't prevent future CI runs from failing.
+# Also call Stop-BuildProcesses
+function Stop-VSProcesses() {
+    Get-Process devenv -ErrorAction SilentlyContinue | kill
```

Comment:
There is an entire list of other processes that VS spawns that 'should' be in here. Still looking for that list so I can update in a future PR.